### PR TITLE
PE-422: Convert PhotoApp to Kotlin

### PR DESCRIPTION
**Pull request by lucianocheng**
Convert PhotoApp to Kotlin per prior discussion on PE-360

@burhanrashid52 let me know if you have time to review this. If not I'll have someone from my team review it.

### CODE REVIEW

**Comment by burhanrashid52** [comment]
@lucianocheng Thanks for the PR. I will be more than happy if someone from your team can review this.

**Comment by lucianocheng** [comment]
> @lucianocheng Thanks for the PR. I will be more than happy if someone from your team can review this.

Sounds good. Could you add @tiendung717 as a collaborator? I will have him review it. Thanks.

File: app/src/main/java/com/burhanrashid52/photoediting/ShapeBSFragment.kt
Comment (lucianocheng) [reply app/src/main/java/com/burhanrashid52/photoediting/ShapeBSFragment.kt:61]: I propose we hold off on this since it falls outside the scope on kotlin refactoring. Left a TODO.

File: app/src/main/java/com/burhanrashid52/photoediting/StickerBSFragment.kt
Comment (lucianocheng) [reply app/src/main/java/com/burhanrashid52/photoediting/StickerBSFragment.kt:52]: Same as above re: refactoring kotlin scope.

File: app/src/main/java/com/burhanrashid52/photoediting/base/BaseActivity.kt
Comment (lucianocheng) [reply app/src/main/java/com/burhanrashid52/photoediting/base/BaseActivity.kt:23]: Same comment as above re: kotlin refactoring scope. Left a TODO.

**Review by lucianocheng — COMMENTED** [review]
@tiendung717 Addressed comments. Let me know. Thanks!

**Review by tiendung717 — APPROVED** [review]
It looks good. Thank you.

**Comment by lucianocheng** [comment]
@burhanrashid52 I think this PR is ready to merge into `kotlin-conversion`. OK to do so?

**Comment by burhanrashid52** [comment]
@lucianocheng Thanks. I am currently looking into the github action issue. Let me try to run the CI for this. Will merge it if it takes more time.

**Comment by burhanrashid52** [comment]
@lucianocheng Can you please the take latest from `kotlin-conversion` and push the changes in this branch?

**Comment by lucianocheng** [comment]
> @lucianocheng Can you please the take latest from kotlin-conversion and push the changes in this branch?

@burhanrashid52 done. Looks like tests are running.

**Pull request merged**
